src/ocf_core_priv.h

```diff
@@ -32,12 +32,65 @@ struct ocf_core_io {
 	/*!< Timestamp */
 };
 
+struct ocf_metadata_uuid {
+	uint32_t size;
+	uint8_t data[OCF_VOLUME_UUID_MAX_SIZE];
+} __packed;
+
+#define OCF_CORE_USER_DATA_SIZE 64
+
+struct ocf_core_meta_config {
+	uint8_t type;
+
+	/* This bit means that object was added into cache */
+	uint32_t added : 1;
+
+	/* Core sequence number used to correlate cache lines with cores
+	 * when recovering from atomic device */
+	ocf_seq_no_t seq_no;
+
+	/* Sequential cutoff threshold (in bytes) */
+	uint32_t seq_cutoff_threshold;
+
+	/* Sequential cutoff policy */
+	ocf_seq_cutoff_policy seq_cutoff_policy;
+
+	/* core object size in bytes */
+	uint64_t length;
+
+	uint8_t user_data[OCF_CORE_USER_DATA_SIZE];
+};
+
+struct ocf_core_meta_runtime {
+	/* Number of blocks from that objects that currently are cached
+	 * on the caching device.
+	 */
+	env_atomic cached_clines;
+	env_atomic dirty_clines;
+	env_atomic initial_dirty_clines;
+
+	env_atomic64 dirty_since;
+
+	struct {
+		/* clines within lru list (?) */
```

Comment:
what about (?) ?

Reply:
This is genuine copy-paste from `src/ocf_cache_priv.h`. Original comments are preserved.